services/submission.js

```diff
@@ -180,6 +181,33 @@ class Submission {
         }, {"$limit": 1}]);
         return (result?.length > 0) ? result[0] : null;
     }
+
+    async getSubmission(params, context){
```

Comment:
Rather than have 4 different identical return statements, let's consolidate theses. You can define each acceptance condition as a boolean variable and then use one if block that checks each like in the pseudocode example below. There should also be an error message thrown if the submission is not returned:
```
ownerCondition = (role is org owner AND submission org matches user org)
submitterCondition = (role is submitter AND current user created submission)
...
if (ownerCondition OR submitterCondition OR ...)
    return submission
else
    throw error